src/azure-cli/azure/cli/command_modules/aro/_aad.py

```diff
@@ -3,109 +3,55 @@
 # Licensed under the MIT License. See License.txt in the project root for license information.
 # --------------------------------------------------------------------------------------------
 
-import datetime
 import time
-import uuid
 
-from azure.cli.core._profile import Profile
-from azure.cli.core.commands.client_factory import configure_common_settings
-from azure.cli.core.azclierror import BadRequestError
-from azure.graphrbac import GraphRbacManagementClient
-from azure.graphrbac.models import ApplicationCreateParameters
-from azure.graphrbac.models import GraphErrorException
-from azure.graphrbac.models import PasswordCredential
-from azure.graphrbac.models import ServicePrincipalCreateParameters
+from azure.cli.command_modules.role import graph_client_factory
+from azure.cli.command_modules.role import GraphError
+
 from knack.log import get_logger
 
 logger = get_logger(__name__)
 
 
+# All methods now return the text based values
+# for consistency and hiding implementation details
 class AADManager:
     def __init__(self, cli_ctx):
-        profile = Profile(cli_ctx=cli_ctx)
-        credentials, _, tenant_id = profile.get_login_credentials(
-            resource=cli_ctx.cloud.endpoints.active_directory_graph_resource_id)
-        self.client = GraphRbacManagementClient(
-            credentials, tenant_id, base_url=cli_ctx.cloud.endpoints.active_directory_graph_resource_id)
-        configure_common_settings(cli_ctx, self.client)
+        self.client = graph_client_factory(cli_ctx)
 
     def create_application(self, display_name):
-        password = uuid.uuid4()
-        start_date = datetime.datetime.utcnow()
-        end_date = datetime.datetime(2299, 12, 31, tzinfo=datetime.timezone.utc)
-
-        app = self.client.applications.create(ApplicationCreateParameters(
-            display_name=display_name,
-            identifier_uris=[],
-            password_credentials=[
-                PasswordCredential(
-                    custom_key_identifier=str(start_date).encode(),
-                    start_date=start_date,
-                    end_date=end_date,
-                    value=password,
-                ),
-            ],
-        ))
-
-        return app, password
+        request_body = {"displayname": display_name, }
```

Comment:
Please adhere to Graph API spec: https://docs.microsoft.com/en-us/graph/api/resources/application?view=graph-rest-1.0

```suggestion
        request_body = {"displayName": display_name}
```